client/my-sites/checkout/index.js

```diff
@@ -193,7 +193,7 @@ export default function () {
 	);
 
 	// Visiting /checkout without a plan or product should be redirected to /plans
-	page( '/checkout', isEnabled( 'jetpack-cloud/connect' ) ? '/plans' : '/pricing' );
+	page( '/checkout', '/plans' );
```

Comment:
I believe that #47770 introduced a Calypso Blue bug here. Before tha patch, `/checkout` would redirect to `/plans` in Calypso Blue. After that patch, the redirect was to `/pricing` (because the feature flag is `false`). Here I'm reverting back to `/plans`.

Reply:
I've seen that happen when trying to visit checkout for non-existent sites and I was wondering why it was happening.